indico/modules/events/layout/controllers/layout.py

```diff
@@ -65,12 +69,50 @@ def _process_lecture_meeting(self):
             return redirect(url_for('.index', self.event_new))
         return WPLayoutEdit.render_template('layout_meeting_lecture.html', self._conf, form=form, event=self.event_new)
 
+    def _make_weeks_view_form(self):
+        form_class = type(b'WeeksViewForm', (IndicoForm,), {})
+        fields = theme_settings.themes['indico-weeks-view']['arguments']
```

Comment:
And how can I get the currently selected theme ? In `views/_init_.py` we get the view from request: `view = request.args.get('view')`   and then use get theme like this : `theme = theme_settings.themes[get_theme(event, view)[0]]` but I dont know how to get the current theme in this context